src/targets/awsLambdaLayer.ts

```diff
@@ -143,56 +141,194 @@ export class AwsLambdaLayerTarget extends BaseTarget {
+        } else {
+          logger.info(
+            `${runtime.name}: ${publishedLayers.length} layers published.`
+          );
+        }
+
+        // Base directory for the layer files of the current runtime.
+        const runtimeBaseDir = path.posix.join(
+          directory,
+          this.AWS_REGISTRY_DIR,
+          runtime.name
+        );
+        if (!fs.existsSync(runtimeBaseDir)) {
+          logger.warn(
+            `Directory structure for ${runtime.name} is missing, skipping file creation.`
+          );
+          return;
+        }
+
+        const regionsVersions = publishedLayers.map(layer => {
+          return {
+            region: layer.region,
+            version: layer.version.toString(),
+          };
+        });
+
+        // Common data specific to all the layers in the current runtime.
+        const runtimeData = {
+          canonical: layerManager.getCanonicalName(),
+          sdk_version: version,
+          account_number: getAccountFromArn(publishedLayers[0].arn),
+          layer_name: this.config.layerName,
+          regions: regionsVersions,
+        };
+
+        const baseFilepath = path.posix.join(
+          runtimeBaseDir,
+          this.BASE_FILENAME
+        );
+        const newVersionFilepath = path.posix.join(
+          runtimeBaseDir,
+          `${version}.json`
+        );
+
+        if (!fs.existsSync(baseFilepath)) {
+          logger.warn(`The ${runtime.name} base file is missing.`);
+          fs.writeFileSync(newVersionFilepath, JSON.stringify(runtimeData));
+        } else {
+          const baseData = JSON.parse(fs.readFileSync(baseFilepath).toString());
+          fs.writeFileSync(
+            newVersionFilepath,
+            JSON.stringify({ ...baseData, ...runtimeData })
+          );
+        }
+
+        createVersionSymlinks(runtimeBaseDir, version, newVersionFilepath);
+        logger.info(`${runtime.name}: created files and updated symlinks.`);
+      })
+    );
   }
+}
 
-  /**
-   * Adds to a layer usage permissions to other accounts.
-   * @param lambda The lambda service object.
-   * @param layerPermissionData Details of the layer and permissions to be set.
-   */
-  public addAwsLayerPermissions(
-    lambda: Lambda,
-    layerPermissionData: AddLayerVersionPermissionRequest
-  ): Promise<any> {
-    return lambda.addLayerVersionPermission(layerPermissionData);
+/**
+ * Creates symlinks to the new version file, and updates previous ones if needed.
+ * @param directory The directory where symlinks will be created.
+ * @param version The new version to be released.
+ * @param versionFilepath Path to the new version file.
+ */
+function createVersionSymlinks(
+  directory: string,
+  version: string,
+  versionFilepath: string
+): void {
+  const latestVersionPath = path.posix.join(directory, 'latest.json');
+  if (fs.existsSync(latestVersionPath)) {
+    const previousVersion = fs
+      .readlinkSync(latestVersionPath)
+      .split('.json')[0];
```

Comment:
```js
const [previousVersion] = fs.readlinkSync(latestVersionPath).split('.json', 1);
```

May consider using [`path.parse()`](https://nodejs.org/api/path.html#path_path_parse_path) or similar path methods too. I checked but didn't find anything that gives you the full path sans the extension but not sure if that's your real intention here. Looks like `path.basename(latestVersionPath, '.json')` would fit here.